docs/deployment/kubernetes.md

```diff
@@ -0,0 +1,137 @@
+# Kubernetes Deployment
+
+## AnyCable-Go
+
+anycable-go can be easily deployed to your Kubernetes cluster using Helm and [our official Helm chart][anycable-helm].
+
+ 1. Add it as a dependency to your main application:
+
+    ```yaml
+    # Chart.yaml
+    dependencies:
+    - name: anycable-go
+      version: 0.2.4
+      repository: https://helm.anycable.io/
+    ```
+
+    Check latest Helm chart version at [github.com/anycable/anycable-helm/releases](https://github.com/anycable/anycable-helm/releases).
+
+ 1. And then configure it in your application values within `anycable-go` namespace:
```

Comment:
What namespace is meant here? Should it be a subdirectory?

Reply:
Here "namespace" is a section in yaml file with values. It must be named `anycable-go` (equals to the chart name)